drivers/hmis/app/graphql/mutations/create_form_definition.rb

```diff
@@ -24,10 +23,105 @@ def resolve(input:)
       )
 
       validation_errors = definition.validate_json_form
+      # rubocop:disable Style/IfUnlessModifier
+      if Hmis::Form::Definition.with_role(definition.role).where(identifier: definition.identifier).exists?
+        validation_errors << HmisErrors::Error.new(:identifier, :invalid, message: 'is not unique. Please choose another identifier')
+      end
+      # rubocop:enable Style/IfUnlessModifier
```

Comment:
This is starting to duplicate with https://github.com/greenriver/hmis-warehouse/pull/4531#discussion_r1675943067 -- @martha do you mind if I just pull that and implement the validation here? (since it's not really related to super-admin)

Reply:
Yep this sounds good!

Reply:
done 👍